### ISSUE TITLE
Prepare should no-op if chef already installed

### ISSUE THREAD

**Issue by aaronjensen** [matschaffer/knife-solo#449]
It'd be great if `knife solo prepare` was idempotent. It could maybe check the version of chef to see if there's an update needed....

**Comment by matschaffer** [comment]
Makes sense. Though we may still want a way to force it if for some reason
chef needs to get reinstalled.

On Tuesday, September 1, 2015, Aaron Jensen <notifications@github.com>
wrote:

> It'd be great if knife solo prepare was idempotent. It could maybe check
> the version of chef to see if there's an update needed....
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matschaffer/knife-solo/issues/449>.
>


-- 

-Mat

matschaffer.com
